hhuijser/liferay-portal
# LPS-78911

To test:

1. Remove lines [sourcechecks.xml#L49-L58](https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/sourcechecks.xml#L49-L58)
2. Add the code below to [source-formatter.properties](https://github.com/liferay/liferay-portal/blob/master/source-formatter.properties)
3. Run formatSource in the [modules/core/petra](https://github.com/liferay/liferay-portal/tree/master/modules/core/petra)

```
override.bnd.exports.check.allowed.export.package.dir.name=\
    modules/apps/foundation/petra/petra-content,\
    modules/apps/foundation/petra/petra-doulos,\
    modules/apps/foundation/petra/petra-encryptor,\
    modules/apps/foundation/petra/petra-io-delta,\
    modules/apps/foundation/petra/petra-json-web-service-client,\
    modules/apps/foundation/petra/petra-log4j,\
    modules/apps/foundation/petra/petra-mail,\
    modules/apps/foundation/petra/petra-model-adapter,\
    modules/apps/foundation/petra/petra-salesforce-client,\
    modules/apps/foundation/petra/petra-xml
```
